website_multi_theme/models/website.py

```diff
@@ -125,15 +126,21 @@ def _multi_theme_activate(self):
         assets_pattern = self.env.ref("website_multi_theme.assets_pattern")
         layout_pattern = self.env.ref("website_multi_theme.layout_pattern")
         for website in self:
-            # Websites without multi theme need to clean their previous views
             if not website.multi_theme_id:
-                _logger.info(
-                    "Deleting multi website theme views for %s: %s",
-                    website.display_name,
-                    website.multi_theme_view_ids,
+                default_theme = self.env.ref(
+                    'website_multi_theme.theme_default'
                 )
-                website.multi_theme_view_ids.unlink()
-                continue
+                if not default_theme:
```

Comment:
If I'm not wrong, this `if` will never pass because the above call to `ref()` doesn't have a `raise_if_not_found=False`.

OTOH I'd use the try-except-else option instead.